plasmapy/analysis/time_series/running_moments.py

```diff
@@ -17,7 +17,7 @@
 _run_moment_tuple = namedtuple("Running_Moment", ["run_moment", "time"])
 
 
-def running_mean(signal, radius):
+def running_mean(signal, radius: int):
     """
```

Comment:
Is the radius allowed to be a float?

Reply:
In this case, `radius` refers to the number of grid points to use in each direction when creating the running mean, so it does need to be an `int`.  This confused me a bit too, so I think this is going to be a place where the type hint is helpful.